C#/C#.sublime-syntax

```diff
@@ -51,6 +51,8 @@ variables:
   name_normal: '{{start_char}}{{other_char}}*\b'
   cap_name: '(\p{Lu}{{other_char}})'
 
+  method_param_type_modifier: \b(out|ref|this|params|in)\s+
```

Comment:
Is there a good way to convert to a non-capturing group and drop the `\s+`? I think there is already a place where this has made a scoping mistake...